doc/classes/@GlobalScope.xml

```diff
@@ -67,7 +67,7 @@
 				Returns the arc tangent of [code]x[/code] in radians. Use it to get the angle from an angle's tangent in trigonometry: [code]atan(tan(angle)) == angle[/code].
```

Comment:
Just catched my eye (you might wanna fix it in here too): note that

> [code]atan(tan(angle)) == angle[/code]

isn't true in general, it's true only for `angle` in [-PI / 2, PI / 2] range.